apps/src/panels/types.ts

```diff
@@ -15,7 +15,10 @@ export type PanelLayout =
 export interface Panel {
   imageUrl: string;
   text: string;
+  key: string;
   nextUrl?: string;
   layout?: PanelLayout;
-  key: string;
+  dark?: boolean;
```

Comment:
Thinking ahead a little bit - should we move this to level properties (and rename `theme`)? I can see per-level theme configuration being really powerful for labs that support it, and we do have initial support for theming in DSCO components now.

Reply:
I think that could make sense longer term.  For the next week of creative exploration, I'd prefer to keep them per-panel so that we can see if different features make sense on different panels.